src/game/audio/sfx.rs

```diff
@@ -28,13 +25,8 @@ pub enum Sfx {
     Step,
 }
 
-fn random_step() -> &'static str {
-    [
-        "audio/sfx/step1.ogg",
-        "audio/sfx/step2.ogg",
-        "audio/sfx/step3.ogg",
-        "audio/sfx/step4.ogg",
-    ]
-    .choose(&mut rand::thread_rng())
-    .unwrap()
+fn random_step(sfxs: &SfxAssets) -> &Handle<AudioSource> {
```

Comment:
This should be a methods on `SfxAssets` :)

Reply:
It seems weird to me to couple in-game SFX logic to the SFX asset struct. IMO this would be a bad precedent.